src/projects.md

```diff
@@ -1,23 +1,102 @@
+
+The following projects aren’t currently being worked on by a project team and are considered dormant. If you're interested in resuming one of these projects, you'll need to start by updating or completing a new [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing) (most of these projects were developed prior to our current project development process and will need to go through the new process to resume).
+
+<div id="projects-inactive">
+{% for project in site.data.inactive_projects %}
+  {% include project.html %}
+{% endfor %}
+</div>
+
+---
+
+{% include linkedHeading.html heading="Decommissioned projects" level=2 %}
+
+Decommissioned projects are projects that the Steering Committee has formally reviewed and deemed no longer a good fit for OpenOakland based on our 2020 project evaluation pilot. These projects may not be reinstated without submitting a new [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing) that substantively addresses the original reasons for discontinuation. Project briefs that are declined by the Steering Committee twice may not be resubmitted without substantive changes.
+
+<div id="projects-decommissioned">
+{% for project in site.data.decommissioned_projects %}
+  {% include project.html %}
+{% endfor %}
+</div>
+
+---
+
+{% include linkedHeading.html heading="Becoming an OpenOakland project" level=2 %}
+
+In response to member feedback that has resulted in an ongoing effort to address issues of equity and inclusion in our work, OpenOakland is reevaluating how our projects are vetted, adopted, and developed. This is an ongoing pilot that we continue to iterate on in an effort to ensure that projects serve their intended communities, consider potential unintended consequences, and foster greater inclusion of community voices—particularly those from underrepresented and underserved Oakland communities.
+
+If you have a new idea for an OpenOakland project:
+
+1. **Fill out the [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing)**. We encourage you to join our [Slack workspace](http://slack.openoakland.org/) and share your draft with our membership, so we can collaborate together as you develop your idea.
+
+2. **Submit your draft brief to the #leadership channel** on Slack for formal consideration. Provided your brief is submitted at least two weeks in advance, it will be reviewed at the next Steering Committee meeting (a group of elected leadership and existing project reps), and you'll get some initial feedback and be asked to make adjustments accordingly.
+
+3. **Make any requested adjustments** based on the Steering Committee's feedback and resubmit the final brief.
+
+Once your final brief is submitted, the Steering Committee will hold a formal vote to approve or decline the project.
+
+### What makes a good project?
```

Comment:
This might be good to have as the top section. It introduces what we value as organization and could help the reader understand what to expect as they scroll through the lists.

Reply:
I moved it up after "active projects," as it's a little long to go at the top. I actually think it deserves its own page, but I'm going to open a separate issue for that. I'd like to get this info published ASAP.